servers/jax-rs-tests/src/main/java/org/projectnessie/jaxrs/tests/AbstractRest.java

```diff
@@ -60,6 +65,10 @@ public NessieApiV1 getApi() {
 
   @AfterEach
   public void tearDown() throws Exception {
+    // Cannot use @ExtendWith(SoftAssertionsExtension.class) + @InjectSoftAssertions here, because
+    // of Quarkus class loading issues. See https://github.com/quarkusio/quarkus/issues/19814
+    soft.assertAll();
```

Comment:
Will this prevent test branch cleanup (below) if it throws? Could we put it in a separate `@AfterEach` method instead?

Reply:
Ah, good catch

Reply:
try-finally works, too